### ISSUE TITLE
Operators to be implemented/fixed in MXNet Backend

### ISSUE THREAD

**Issue by sandeep-krishnamurthy** [deep-learning-tools/keras#5]
Below is the checklist of operators to be implemented or revisited for fixing issues in MXNet backend for Keras:
- [ ] Batchnorm
- [ ] BatchDot 
- [ ] Binary CrossEntropy
- [ ] Categorical CrossEntropy
- [ ] logsumexp
- [ ] Sparse operators
- [ ] bias_add
- [ ] temporal_padding - MXNet do not support 3D Tensor padding
- [ ] conv1d/2d/3d
- [ ] rnn
